dbt_copilot_helper/commands/dns.py

```diff
@@ -501,20 +512,21 @@ def _get_subdomains_from_env_manifests(environment, manifests):
                 click.echo(
                     click.style("Checking file: ", fg="cyan") + click.style(manifest, fg="white"),
                 )
-                click.secho("Domains listed in manifest file", fg="cyan", underline=True)
-
-                aliases = [
-                    domain["http"]["alias"]
-                    for env, domain in conf["environments"].items()
-                    if env == environment
-                ]
-
-                click.secho(
-                    "  " + "\n  ".join(aliases),
-                    fg="yellow",
-                    bold=True,
-                )
-                subdomains.extend(aliases)
+                aliases = _get_domain_aliases(conf, environment)
+                if not len(aliases):
+                    click.echo(
+                        click.style(
+                            "Misconfigured: no domain or alias present, skipping...", fg="cyan"
```

Comment:
It may not be a misconfiguration, e.g. https://github.com/uktrade/demodjango-deploy/blob/main/copilot/celery-worker/manifest.yml#L44

Maybe just...
```suggestion
                            f"No http.alias present for {environment} environment in {manifest}, skipping...", fg="cyan"
```